### ISSUE TITLE
[SF] [Toast] [A11y] Toast message is not announced by screen reader

### ISSUE THREAD

**Issue by 2bno1** [SAP/ui5-webcomponents#3841]
### **Bug Description**
Toast message is not announced by NVDA screen reader

### **Expected Behavior**
Toast message should be announced by screen reader

### **Steps to Reproduce**
1. Download and install https://www.nvaccess.org/download/
2. Open NVDA screen reader
3. Open https://hue07.codesandbox.io/ in browser
4. Click 'Basic Toast' button
5. Verify whether the toast message is announced by NVDA screen reader

### **Isolated Example**
https://codesandbox.io/s/ui5-webcomponents-forked-hue07?file=/index.html

### **Context**
- UI5 Web Components version: 0.31.16
- OS/Platform: macOS
- Browser: Chrome
- Affected component: Toast

### **Log Output / Stack Trace / Screenshots**
![image](https://user-images.githubusercontent.com/1751846/132292046-a9db7351-115c-4e9b-b46a-15907110d78d.png)

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SuccessFactors

**Comment by gmkv** [comment]
Hi @2bno1 
I can't reproduce this issue myself - NVDA announces the Toast, and so does JAWS.
Can you check again?

Regards,
Georgi

**Comment by gmkv** [comment]
Hi @SAP/ui5-webcomponents-topic-rl 

There's an accessibility issue with the Toast not being read out with the latest version of NVDA - v2021.1.
The issue is not reproducible with NVDA v2020.4.
Sample: https://cdhqh.csb.app/  -  [(code)](https://codesandbox.io/s/ui5-webcomponents-forked-cdhqh?file=/index.html)

Kind regards,
Georgi

**Comment by ericchan1983** [comment]
Hi @gmkv,

After testing, the toast message can be announced in latest version of NVDA - v2021.1 by adding the role="alert" for "Toast" like this:
<ui5-toast id="wcToastBasic" duration="10000" role="alert">
    This text should be announced by screen reader
</ui5-toast >
But the expected behavior should be "Alert This text should be announced by screen reader", not just "This text should be announced by screen reader". The screen reader should announces the Alert for the toast just like UI5 demo page below.
![image](https://user-images.githubusercontent.com/1614528/133573063-e1d8ee9a-8c8d-410a-b5f6-af7456626b3d.png)


Regards,
Eric Chen

**Comment by ilhan007** [comment]
Fixed with 0.31.18
